application/templates/datamanager/flagged-resources-start.html

```
@@ -0,0 +1,120 @@
+      element: datasetContainer,
+      id: "dataset-autocomplete",
+      name: "dataset-autocomplete-visible",
+      defaultValue: datasetValue,
+      minLength: 2,
+      confirmOnBlur: false,
+      autoselect: true,
+      displayMenu: "overlay",
+      placeholder: "Type dataset name...",
+      source: function(query, populateResults) {
+        fetch(`{{ url_for('datamanager.dashboard_get') }}?autocomplete=${encodeURIComponent(query)}`)
+          .then((response) => response.json())
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)